gradle.properties

```diff
@@ -3,7 +3,7 @@ org.gradle.daemon=false
 # Mod
 mod_id=ftblibrary
 readable_name=FTB Library
-mod_version=2101.1.25
+mod_version=2101.1.26
```

Comment:
Keep version bumps in their own commits, it messes with cherry-picking otherwise.